ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManagerImpl.java

```diff
@@ -459,6 +459,10 @@ public void showDevOptionsDialog() {
         new DevOptionHandler() {
           @Override
           public void onOptionSelected() {
+            if (mDevSettings.isHotModuleReplacementEnabled()) {
+              Toast.makeText(mApplicationContext, "Dev mode and HRM cannot be both enabled at once. Disabling HMR...", Toast.LENGTH_LONG).show();
```

Comment:
I'm confused by this message. HMR only works in Dev mode, so they both need to be enabled. When exactly is this shown?

Also, it says HRM, not HMR.

Reply:
Sorry, I worked on this late at night and the message was a nonsense.